pkg/nuctl/command/get.go

```diff
@@ -97,14 +99,17 @@ func newGetFunctionCommandeer(getCommandeer *getCommandeer) *getFunctionCommande
 				return nil
 			}
 
-			// render the functions
-			return commandeer.renderFunctions(functions, commandeer.getOptions.Format, cmd.OutOrStdout())
+			// render the functions, if error occurs return it, else return nil
+			return commandeer.renderFunctions(functions, commandeer.getOptions.MatchCriterias[0].Format, cmd.OutOrStdout())
```

Comment:
Same comment as above - the Format field is common to all MatchCriteria structs so it should be moved out of them and to the root of the GetOptions struct.